blackboard/gui/result_pane.py

```diff
@@ -0,0 +1,30 @@
+import gi
+gi.require_version('Gtk', '3.0')
+
+from gi.repository import Gtk, Gio, Gdk  # noqa
+
+
+class result_pane(Gtk.Grid):
+    def __init__(self):
+        Gtk.Grid.__init__(self)
+        self.__init_lb_list()
+
+    def __init_lb_list(self):
```

Comment:
You're right!
I've changed the "add_label" method and I've eliminated the "__init_lb_list" method.
See commit d3bd51a2b6ae4e9d234e16c2c0e3ad36a6aa3ecf